l10n_it_edi_related_document/views/l10n_it_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_invoice_form_related_document" model="ir.ui.view">
+        <field name="name">account.move.form.related.document</field>
+        <field name="model">account.move</field>
+        <field name="priority">20</field>
+        <field name="inherit_id" ref="l10n_it_edi.account_invoice_form_l10n_it" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='electronic_invoicing']" position="after">
+                <page
+                    string="Related Document"
+                    name="related_document"
+                    invisible="move_type not in ('out_invoice', 'out_refund', 'in_invoice', 'in_refund') or country_code != 'IT'"
+                >
+                    <field name="related_document_ids">
+                        <list editable="bottom">
+                            <field name="type" />
+                            <field name="name" />
+                            <field name="lineRef" />
+                            <field name="date" />
+                            <field name="numitem" />
+                            <field name="code" />
+                            <field name="cig" />
+                            <field name="cup" />
+                        </list>
+                    </field>
+                </page>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="account_invoice_hide_document" model="ir.ui.view">
+        <field name="name">account.move.hide.document.type</field>
+        <field name="model">account.move</field>
+        <field name="priority">99</field>
```

Comment:
Questa priorità così alta serviva perché prima delle ultime modifiche c'era un `replace`, come mai serve ancora?

Volendo questo `xpath` si potrebbe includere nella vista precedente, ma non vedo problemi a lasciarlo in una vista a sé.